src/pages/Login.tsx

```diff
@@ -24,6 +24,7 @@ import User from '../Models/User';
 async function handleSubmit(email: string, password: string) {
     if (await User.login(email, password)) {
         alert('login success');
+        window.history.replaceState({}, '', '/tab1'); //TODO Redirect use to the homepage
```

Comment:
@MrClean-1 @lemohammed: Does this need to redirect to `/home` or `/landing`?

Reply:
That needs to be to /home. Let me change it.
I had something similar in App.tsx